libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -415,4 +473,109 @@ public void onReceive(Context context, Intent intent) {
             }
         }
     }
+
+    private void presentBiometric() {
```

Comment:
The reset of the class is biometric code that is extremely similar to what is in `ScreenLockActivity` so I will consider creating a single biometric utility class they both use.